indico/web/client/js/react/components/category_navigator/view.jsx

```diff
@@ -0,0 +1,350 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Dialog from 'indico/react/components/Dialog';
+import {Translate, PluralTranslate, Param} from 'indico/react/i18n';
+import 'indico/custom_elements/ind_with_tooltip';
+import 'indico/custom_elements/ind_with_toggletip';
+
+import './style.scss';
```

Comment:
"Famous last words" ;)

But indeed, I see you have everything under `#category-nav-dialog` which is clear enough. Otherwise classes like `.category-list-section`, `.category-search-results` or `.category-list-container` are not necessarily unique - for example such a class name could also be used on a category display page (even though people should of course check for conflicts anyway:))